etc/adminhtml/system.xml

```diff
@@ -138,6 +138,11 @@
                         <field id="akeneo_connector/justbetter/defaultstorevalues">1</field>
                     </depends>
                 </field>
+                <field id="remove_redundant_eav" translate="label" type="select" sortOrder="160" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Remove Redundant EAV</label>
```

Comment:
What are Redundant Eav attribute values in our context? Are they unused attribute values?
Maybe we could clear that up in the comment under here

Reply:
Redundant EAV attribute values are created when the family (attribute set) of a product is changed in Akeneo. Attributes that were previously associated with the product, but no longer appear in the new attribute set, still remain in the database.

This option cleans up these redundant values so that they no longer remain in the database. This prevents unnecessary or irrelevant data from being left behind.

> For Rapidez projects, where product specifications are displayed directly from the database on the PDP, it could happen that specifications were displayed that were no longer relevant to the product's current family. This clean-up fixes that problem.

I will also add some additional explanation to the configuration option so that it is clearer exactly what it does.